{"created": 1760612558.8405602, "duration": 0.036544084548950195, "exitcode": 1, "root": "/Users/ehtesham/Developer/aadhaar-api-test-poc", "environment": {}, "summary": {"error": 6, "total": 6, "collected": 6}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "tests/test_aadhaar_api_v5.py", "type": "Module"}]}, {"nodeid": "tests/test_aadhaar_api_v5.py", "outcome": "passed", "result": [{"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_forbidden_access", "type": "Function", "lineno": 57}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_invalid_consent", "type": "Function", "lineno": 68}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_missing_field", "type": "Function", "lineno": 80}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_success", "type": "Function", "lineno": 91}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_verify_aadhaar_missing_field", "type": "Function", "lineno": 104}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_verify_aadhaar_success", "type": "Function", "lineno": 113}]}], "tests": [{"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_forbidden_access", "lineno": 57, "outcome": "error", "keywords": ["test_demographics_forbidden_access", "test_aadhaar_api_v5.py", "tests", "aadhaar-api-test-poc", ""], "setup": {"duration": 0.0001854998990893364, "outcome": "failed", "longrepr": "file /Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py, line 58\n  def test_demographics_forbidden_access(session):\nE       fixture 'session' not found\n>       available fixtures: _session_event_loop, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, event_loop, event_loop_policy, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tests/test_aadhaar_api_v5.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py:58"}, "teardown": {"duration": 6.733275949954987e-05, "outcome": "passed"}}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_invalid_consent", "lineno": 68, "outcome": "error", "keywords": ["test_demographics_invalid_consent", "test_aadhaar_api_v5.py", "tests", "aadhaar-api-test-poc", ""], "setup": {"duration": 7.10831955075264e-05, "outcome": "failed", "longrepr": "file /Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py, line 69\n  def test_demographics_invalid_consent(session):\nE       fixture 'session' not found\n>       available fixtures: _session_event_loop, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, event_loop, event_loop_policy, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tests/test_aadhaar_api_v5.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py:69"}, "teardown": {"duration": 4.6292319893836975e-05, "outcome": "passed"}}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_missing_field", "lineno": 80, "outcome": "error", "keywords": ["test_demographics_missing_field", "test_aadhaar_api_v5.py", "tests", "aadhaar-api-test-poc", ""], "setup": {"duration": 7.200054824352264e-05, "outcome": "failed", "longrepr": "file /Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py, line 81\n  def test_demographics_missing_field(session):\nE       fixture 'session' not found\n>       available fixtures: _session_event_loop, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, event_loop, event_loop_policy, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tests/test_aadhaar_api_v5.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py:81"}, "teardown": {"duration": 4.099961370229721e-05, "outcome": "passed"}}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_demographics_success", "lineno": 91, "outcome": "error", "keywords": ["test_demographics_success", "test_aadhaar_api_v5.py", "tests", "aadhaar-api-test-poc", ""], "setup": {"duration": 5.3416937589645386e-05, "outcome": "failed", "longrepr": "file /Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py, line 92\n  def test_demographics_success(session):\nE       fixture 'session' not found\n>       available fixtures: _session_event_loop, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, event_loop, event_loop_policy, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tests/test_aadhaar_api_v5.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py:92"}, "teardown": {"duration": 5.070772022008896e-05, "outcome": "passed"}}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_verify_aadhaar_missing_field", "lineno": 104, "outcome": "error", "keywords": ["test_verify_aadhaar_missing_field", "test_aadhaar_api_v5.py", "tests", "aadhaar-api-test-poc", ""], "setup": {"duration": 6.070826202630997e-05, "outcome": "failed", "longrepr": "file /Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py, line 105\n  def test_verify_aadhaar_missing_field(session):\nE       fixture 'session' not found\n>       available fixtures: _session_event_loop, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, event_loop, event_loop_policy, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tests/test_aadhaar_api_v5.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py:105"}, "teardown": {"duration": 3.600027412176132e-05, "outcome": "passed"}}, {"nodeid": "tests/test_aadhaar_api_v5.py::test_verify_aadhaar_success", "lineno": 113, "outcome": "error", "keywords": ["test_verify_aadhaar_success", "test_aadhaar_api_v5.py", "tests", "aadhaar-api-test-poc", ""], "setup": {"duration": 5.3165480494499207e-05, "outcome": "failed", "longrepr": "file /Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py, line 114\n  def test_verify_aadhaar_success(session):\nE       fixture 'session' not found\n>       available fixtures: _session_event_loop, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, cov, doctest_namespace, event_loop, event_loop_policy, include_metadata_in_junit_xml, json_metadata, metadata, monkeypatch, no_cover, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, tests/test_aadhaar_api_v5.py::<event_loop>, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, unused_tcp_port, unused_tcp_port_factory, unused_udp_port, unused_udp_port_factory\n>       use 'pytest --fixtures [testpath]' for help on them.\n\n/Users/ehtesham/Developer/aadhaar-api-test-poc/tests/test_aadhaar_api_v5.py:114"}, "teardown": {"duration": 5.8292411267757416e-05, "outcome": "passed"}}]}